docs/src/global/exampleFrame.jsx

```diff
@@ -0,0 +1,79 @@
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import PropTypes from 'prop-types';
+import React, {
+    useEffect,
+    useRef,
+    useState,
+} from 'react';
+import ReactDOM from 'react-dom';
+import ExampleFramed from './exampleFramed';
+
+const propTypes = {
+    children: PropTypes.node,
+    height: PropTypes.number,
+    maxWidth: PropTypes.number, // eslint-disable-line react/no-unused-prop-types
```

Comment:
Sigh.  Too bad ESLINT isn't smart about props used in the styles function.  But it's probably difficult to detect that.

Reply:
Yeah, maybe in the future someone will fix this.

Reply:
Resolving this to de-clutter the PR.